SuffolkLITLab/docassemble-MA209AProtectiveOrder
# Fix #52, cannot edit plaintiff age

Test:
1. Set plaintiff age to be over 18
1. Get past 'basic information'
1. Tap on 'basic information' section on left side
1. Note text next to birthdate editing button (over 18)
1. Choose to edit birthdate
1. Edit button takes you to plaintiff birthdate page
1. Edit age to under 18
1. Tap 'next'
1. See review page text has changed (under 18)
